core-web/libs/block-editor/src/lib/extensions/bubble-link-form/components/suggestion-page/suggestion-page.component.html

```diff
@@ -0,0 +1,25 @@
+<div *ngIf="items.length; else loading ? loadingBlock : emptyBlock">
+    <dot-suggestion-list #list>
+        <dot-suggestions-list-item
+            *ngFor="let item of items; let i = index"
+            [command]="item.command"
+            [index]="i"
+            [label]="item.label"
+            [url]="item.icon"
+            [data]="item.data"
+            [page]="true"
+        >
+        </dot-suggestions-list-item>
+    </dot-suggestion-list>
+</div>
+
+<ng-template #loadingBlock>
+    <dot-suggestion-loading-list *ngIf="loading"></dot-suggestion-loading-list>
+</ng-template>
+
+<ng-template #emptyBlock>
+    <div class="empty">
+        <p [innerHTML]="title"></p>
+        <button (mousedown)="handleBackButton($event)">Back</button>
+    </div>
```

Comment:
I think we can extract this to a component, only if we use it in more than once place.